README.md

```diff
@@ -1,17 +1,15 @@
 # Fork Awesome
 ### A fork of the iconic font and CSS toolkit
 
-[![npm-badge]][npm-link] [![all-contrib]](CONTRIBUTORS.md) [![jsdeliver-badge]][jsdeliver-link] [![cdnjs-badge]][cdnjs-link] [![build-status-badge]][build-status-link]
+[![npm-badge]][npm-link] [![all-contrib]](CONTRIBUTORS.md) [![jsdeliver-badge]][jsdeliver-link] [![cdnjs-badge]][cdnjs-link] [![build-status-badge]][build-status-link] [![chat-badge]][chat-link]
 
-Fork Awesome is a full suite of 796 pictographic icons for easy scalable vector graphics on websites, originally created by [Dave Gandy](https://twitter.com/davegandy) and now maintained by a community.
+**Fork Awesome is a suite of 796 pictographic and brand icons for easy, scalable vector graphics on websites and beyond.**
 
-Following concerns regarding [the development of Font Awesome](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956), the PR Freeze since Oct 2016 and the direction [Fort Awesome](https://fortawesome.com/) is taking with the version 5.0 of their project, we are forking Font Awesome (4.7), in order to build on this amazing tool Dave Gandy has given us, while at the same time allowing this project to be run by a distributed community of contributors.
```

Comment:
Since we are revamping this section, can we also move the link pointers to the bottom of the section? Like it is done for the badges. That way the content becomes readable and _un-cluttered_ from links sprinkled over the place.

Reply:
Not against that.

Reply:
Actually, I've had a change of thought on this :sweat_smile: I agree it disturbs the flow of reading raw a little bit, but having to maintain currency with links stored elsewhere is also irritating. I could see it making more sens if we were referring to the same URL multiple times, but al our URL's are unique...

I would even suggest moving the badge URLs to the badges, but happy to leave.

Reply:
My suggestion was not to move the links all the way to the bottom of the page, but to the bottom of the section ( that is, right above the **How to use** heading ). The links would still be in view.

If you feel strongly about not moving the links, then okay. But let us not move the badge links.